server/client-config/routes.test.js

```diff
@@ -36,4 +62,10 @@ describe('Client config handler', () => {
     const response = await request.get('/config.js').expect(200)
     expect(response.text).toMatchSnapshot()
   })
+
+  it('returns something pointing ping', async () => {
+    const request = makeApp()
+    const response = await request.get('/ping').expect(200)
+    expect(response.text).toMatchSnapshot()
```

Comment:
Snapshots are useful for matching against complex values which are expected to change and need updating. (For instance the client config object in the test above is likely to change in future as more values are added.) Here it would be better to use `.toEqual` as that puts the expected value right in the test.